manage_breast_screening/participants/models.py

```diff
@@ -16,7 +16,7 @@ class Participant(BaseModel):
     ethnicity = models.TextField()
     address = models.TextField()
     risk_level = models.TextField()
-    extra_needs = models.JSONField(null=False, default=[])
```

Comment:
this is a common python bug - keyword arguments are created once at definition time, so instances would share a reference to a single list instance